### ISSUE TITLE
how to install it ? 

### ISSUE THREAD

**Issue by teknosains** [sergeyk/selective_search_ijcv_with_python#2]
just download or how ?. im trying to use caffe with this module

Thanks

**Comment by roseperrone** [comment]
Yeah, cd into `caffe/python/caffe` and run `git clone git@github.com:sergeyk/selective_search_ijcv_with_python.git`. Caffe will find it, no need to put it on your PYTHONPATH.

**Comment by zirohut** [comment]
@codetrash @roseperrone 
Hi,
I tried to run 
git clone git@github.com:sergeyk/selective_search_ijcv_with_python.git 
but it says : Permission denied (publickey)

Is there another way to install it?

Thanks.

**Comment by roseperrone** [comment]
@zirohut I can clone it without error. Find out if there's something fishy with your github key setup.

**Comment by zirohut** [comment]
@roseperrone 
Thanks.
Another question. Do I have run it in matlab to compile the functions or I can just directly use the python module in r-cnn (caffe) ?
Thanks a lot.

**Comment by roseperrone** [comment]
@zirohut I don't recall what I did. Try running it using python. If you're on mac, you might run into these (solved) issues: see my comment in this thread: https://groups.google.com/forum/#!topic/caffe-users/-UIxoqA5iJ0. Sorry I'm not that helpful. Post another question when you run into an error.

**Comment by zirohut** [comment]
@roseperrone 
Thanks, I understand how it works now.

**Comment by Linzert** [comment]
@zirohut Do you solve your problems?I have the selective_search_ijcv_with_python files in caffe/python/caffe,and runned the demo.m,but when I run python selective_search.py,I got the error message "OSError: [Errno 2] No such file or directory".What should I do?

**Comment by zirohut** [comment]
@Linzert 

I think that's probably because the path setting in the .py file isn't correct. Just check out the py file and modify the path settings.

**Comment by Linzert** [comment]
@zirohut Firstly,thanks for your reply.But in the selective_search.py file,there is not any path only "script_dirname=os.path.abspath(os.path.dirname(**file**))",I donnot know how to set the path.Could you please elaborate it?

**Comment by zirohut** [comment]
@Linzert 
I'm actually getting two different messages in my email and on this github page. "OSError: [Errno 2] No such file or directory" on this github page, and "ImportError: No module named selective_search_ijcv_with_python" in my email.

If it's the OSError, I checked out the selective_search.py file. In the main function, 
image_filenames = [
        script_dirname + '/000015.jpg',
        script_dirname + '/cat.jpg'
    ] \* 4
if the error shows no such file or directory, maybe it's because the dirname for image is not correct?

If it's the import error, you probably want to check the system path settings.

**Comment by Linzert** [comment]
@zirohut Acturally it's the OSError.In the main function the image_filenames is correct and they are default in the selective_search_ijcv_with_python file.I think the error may be in other places but not the image_filenames.

**Comment by sunshineatnoon** [comment]
@zirohut Do you need MATLAB to run this code?

**Comment by zirohut** [comment]
@sunshineatnoon 

I ran demo.m in MATLAB. But I think python would work too.

**Comment by CLTFOREVER** [comment]
@Linzert  mc = "/usr/local/matlab/bin/matlab -nojvm -r \"try; {}; catch; exit; end; exit\"".format(command)
you should set the MATLAB path as your own path  ,on my computer ,my MATLAB path is /usr/local/matlab/bin/matlab

**Comment by arushk1** [comment]
do : 

cd /usr/local/bin/
sudo ln -s /usr/local/MATLAB/R2013a/bin/matlab matlab

**Comment by alviur** [comment]
@CLTFOREVER  Thanks! that worked for me!

**Comment by Lvhhhh** [comment]
@CLTFOREVER  my MATLAB path is "D:\matlab\bin".but i change mc="D:\matlab\bin\matlab" it dos not work." scipy.io.matlab.miobase.MatReadError: Mat file appears to be empty"

**Comment by UpCoder** [comment]
Hi, May i ask a question? Is the selective region dependency with caffe? I use tensorflow, can I use it? If yes, how to install it?
 Thanks
